reference/forms/types/dateinterval.rst

```diff
@@ -0,0 +1,293 @@
+.. index::
+   single: Forms; Fields; DateIntervalType
+
+DateIntervalType Field Type
+===========================
+
+This field type allows the user to modify data that represents a specific
+date interval. It can be rendered as a integer or text input or select tags.
+
+The underlying format of the data can be a ``DateInterval`` object,
+a `RFC 3339`_ duration string (e.g. ``P1DT12H``) or an array.
+
++----------------------+-----------------------------------------------------------------------------------+
+| Underlying Data Type | can be ``DateInterval``, string or array (see the ``input`` option)               |
++----------------------+-----------------------------------------------------------------------------------+
+| Rendered as          | single text box or up to six select, text or integer boxes plus optional checkbox |
++----------------------+-----------------------------------------------------------------------------------+
+| Options              | - `days`_                                                                         |
+|                      | - `placeholder`_                                                                  |
+|                      | - `hours`_                                                                        |
+|                      | - `input`_                                                                        |
```

Comment:
@MisatoTremor When I reviewed this earlier today the diff looked completely broken. Looks like this was related to my browser then or the issue disappeared with your push. So everything's fine here how. :)